src/main/java/org/jenkinsci/test/acceptance/junit/Native.java

```diff
@@ -45,9 +63,34 @@ public void evaluate() throws Throwable {
 
                 private void verifyNativeCommandPresent(Native n) throws IOException, InterruptedException {
                     if (n==null)        return;
+                    
+                    // Checks performed in this annotation only makes sense if Jenkins and Tests are being executed in the same machine
+                    if (!(controller instanceof LocalController)) {
+                        throw new AssumptionViolatedException("Test skipped. Native should be used with a local controller, otherwise it cannot be fully trusted to work as expected.");
+                    }
+                    
+                    List<String> supportedCmds = Lists.newLinkedList();
+                    // Get PATH. Valid for both Unix and Windows
+                    String path = System.getenv("PATH");
```

Comment:
No - just revert this hunk.  I have a much better fix in #84 [`Native.java#47-62`](https://github.com/jtnord/acceptance-test-harness/blob/jenkins-33520/src/main/java/org/jenkinsci/test/acceptance/junit/Native.java#L47-L62)